.github/workflows/test.yml

```diff
@@ -26,20 +26,25 @@ jobs:
       - name: Create LocalGov Drupal project
         run: composer create-project --stability dev localgovdrupal/localgov-project html
 
-      - name: Setup package source for the test target
-        run: |
-          composer --working-dir=html config repositories.1 vcs git@github.com:${{ github.repository }}.git
-          composer global config github-oauth.github.com ${{ github.token }}
-      - name: Extract Git branch name outside of a pull request
+      - name: Save git branch and git repo names to env if not a pull request
         if: github.event_name != 'pull_request'
-        run: echo "GIT_BRANCH=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+        run: |
+          echo "GIT_BRANCH=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+          echo "GIT_REPO=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
 
-      - name: Extract Git branch name from a pull request
+      - name: Save git branch and git repo names to env if is a pull request
         if: github.event_name == 'pull_request'
-        run: echo "GIT_BRANCH=${GITHUB_HEAD_REF}" >> $GITHUB_ENV
+        run: |
+          echo "GIT_BRANCH=${GITHUB_HEAD_REF}" >> $GITHUB_ENV
+          echo "GIT_REPO=${{ github.event.pull_request.head.repo.full_name }}" >> $GITHUB_ENV
+
+      - name: Setup package source and authentication for the test target
+        run: |
+          composer --working-dir=html config repositories.1 vcs git@github.com:${GIT_REPO}.git
+          composer global config github-oauth.github.com ${{ github.token }}
 
-      - name: Grab test target
-        run: composer --working-dir=html require ${{ github.repository }}:"dev-${GIT_BRANCH} as 1.0.x-dev"
+      - name: Grab test target for hardcoded package name 'localgov'
+        run: composer --working-dir=html require localgovdrupal/localgov:"dev-${GIT_BRANCH} as 1.0.x-dev"
```

Comment:
Could you please extract the "localgov" part from the **github** context.  If that doesn't work then we will have to settle for hardcoding.  Something like the following should do:
```
name: Grab test target
run: composer --working-dir=html require localgovdrupal/${{ github.event.repository.name }}:"dev-${GIT_BRANCH} as 1.0.x-dev"
```

I am saying this because we have a [template](https://github.com/localgovdrupal/.github/blob/master/workflow-templates/localgov-drupal-ci.yml) for the test config file.  This template is setup so that it can be added verbatim to any new localgovdrupal repo and it just works.  If we hardcode a value here and someone later updates this file with the template, which happens from time to time as the template evolves, then we will loose the hardcoded value.

FAO @tebb @finnlewis